MultiPoolMiner.ps1

```diff
@@ -114,6 +114,12 @@ Write-Log "Starting MultiPoolMiner® v$Version © 2017-2018 MultiPoolMiner.io"
 if (-not [IO.Path]::GetExtension($ConfigFile)) {$ConfigFile = "$($ConfigFile).txt"}
 if (Test-Path $ConfigFile) {
     Write-Log -Level Info "Using configuration file ($(Resolve-Path $ConfigFile)). "
+    #Set API_ID, API_Key, UserName, Wallet and WorkerName if not passed as command line parameter, so they get properly inherited to the pool files
```

Comment:
Why do these variables need special treatment?

Reply:
Because it does not work otherwise if your config file conteins pool entries like this:
```
    "MiningPoolHub": {
      "User": "$UserName",
      "Worker": "$WorkerName",
      "API_ID": "$API_ID",
      "API_Key": "$API_Key",
      "MinWorker": 1,
    },
```
$UserName, WorkerName etc. remain empty if they are NOT passed by command line.
Without the proposed change all special variables need to be hardcoded in all config items.

Reply:
Surely all the parameters need to be hardcoded anyway. Do you not dump a config file to the hard disk for the users to edit on the first run?

Reply:
A new (empty) config file does not contain any pool info. If someone is adding a pool section as in my example then it is not going to work (without my proposed change).

The same pool section works however if the variables are passed as command line.
So my change basically does not break anything, but it allows inheritance of the special variables.